Improve helm lint documentation clarity (#1637)
helm/helm-www: content/en/docs/helm/helm_lint.md

      --skip-schema-validation    if set, disables JSON schema validation
      --strict                    fail on lint warnings
  -f, --values strings            specify values in a YAML file or a URL (can specify multiple)

>       --with-subcharts            lint subcharts as independent charts (note that without this option, the chart will be validated as a whole, including subcharts as its part)

Reviewer: This document is generated from Helm itself. So are all the files in this directory.

It needs to be changed at https://github.com/helm/helm/blob/ecc4adee692333629dbe6343fbcda58f8643b0ca/cmd/helm/lint.go#L149.

Please update it in the other location.

Reviewer 2: Additionally, the wording is still hazy. Maybe something like this could be better:
```suggestion
      --with-subcharts            lint subcharts separately from the parent chart
```
To be honest, this option's name is already misleading. But that's much harder to change.
